src/Ip.php

```diff
@@ -74,6 +74,37 @@ public static function isValidv6($ip)
     }
 
     /**
+     * Checks if an IP is local
+     *
+     * @param  string  $ip IP
+     * @return boolean     true if the IP is local, otherwise false
+     */
+    public static function isLocal($ip)
+    {
+        if (self::isValidv4($ip)) {
+            return self::match($ip, '10.*.*.*') ||
```

Comment:
Rather than calling `self::match()` repeatedly it might be cleaner to define an array of local IP patterns and loop over them